### ISSUE TITLE
Convert captured faces into local binary patterns

### ISSUE THREAD

**Issue by xu-chris** [n1g1r1/cv-face-extractor#1]
As a user I want to capture faces as local binary patters to get illuminance invariant training sets for better and more stable results throughout different webcams and settings.

**Comment by Quving** [comment]
In my opinion it is not necessary to obtain binary images and train the network afterward with. As far as i know, convolutional neural networks such as VGG take over the part of feature extractions. Especially, since the VGG16 net is used to fine tune our classifier, I suspect that using binary images would increase the accuracy. In addition, binary images does not provide rgb spectrums - so color based recognition will be maybe disabled.

**Comment by xu-chris** [comment]
That's true. Even LBP algorithms are calculating further the histograms than the binarized image. We can close this. Thanks for contribution!
